pact.cabal

```diff
@@ -64,6 +64,22 @@ library pact-prettyprinter
     , ansi-terminal >=0.4
     , prettyprinter >= 1.7
 
+-- -------------------------------------------------------------------------- --
+-- Internal: unsafe functions from base
+--
+-- This is to avoid cluttering production code with
+-- `{-# OPTIONS_GHC -Wno-x-partial #-}` pragmas with base >= 4.20
+
+library unsafe
+  visibility: public
```

Comment:
why public, shouldn't `private` for internal libs be enough?

Reply:
Those modules are also used in chainweb-node from where they were copied.

I guess, I could have moved this library into an independent package -- but it seemed a bit overkill. Still I left them in a separate library so that we can move it more easily to an independent package in case we switch to a monorepo in the future.